README.md

```diff
@@ -38,13 +39,26 @@ NOTE: The browsersync'd URL is ***HTTPS***, not HTTP. ddev-router redirects traf
 EG.
 "External: <http://d9.ddev.site:3000>" => Access on **<https://d9.ddev.site:3000>**
 
+### Note
+
+From version `2.5.0`, this addon uses a "per-project" approach. This will not affect usage as project-level command have higher priority than global commands.
+
+When installing current versions, a warning is displayed if the global command (`~/.ddev/commands/web/browsersync`) is detected. You may safely delete his file.
+
+If you run `ddev browsersync` from multiple local projects and receive a `Error: unknown command "browsersync" for "ddev"`, run the following command to add the command to the project as needed.
```

Comment:
Reframe for the context of a singular project, since it sounds like the person would need to run the command per-project anyway. (Would someone likely be running from multiple projects simultaneously?) Use a colon to pitch to the command we explicitly mention.

```suggestion
If you run `ddev browsersync` from a local project and get `Error: unknown command "browsersync" for "ddev"`, run the following to add the command to the project:
```